pkgs/applications/virtualization/runc/default.nix

```diff
@@ -12,33 +12,29 @@
 , nixosTests
 }:
 
-buildGoPackage rec {
+buildGoModule rec {
```

Comment:
Why are we switching from `buildGoPackage` to `buildGoModule`?
There is some push in nixpkgs to get rid of or reduce the usage of the latter.

Reply:
We're still using the upstream vendor so there isn't a practical difference except that we don't need to deal with GOPATH. This is the only podman related package than hasn't already been changed to `buildGoModule`.

> There is some push in nixpkgs to get rid of or reduce the usage of the latter.

AFAIK `buildGoModule` itself isn't a problem, only when it uses fixed outputs.

Reply:
So we still have to use `buildGoPackage` because of `docker` which needs the build to be compatible with all of its vendored versions of `runc`.